src/m365/todo/commands/list/list-remove.ts

```diff
@@ -114,10 +73,51 @@ class TodoListRemoveCommand extends GraphCommand {
       });
 
       if (result.continue) {
-        await removeList();
+        await this.removeList(args);
       }
     }
   }
+
+  private async getListId(args: CommandArgs): Promise<string> {
+    if (args.options.name) {
+      // Search list by its name
+      const requestOptions: any = {
+        url: `${this.resource}/v1.0/me/todo/lists?$filter=displayName eq '${escape(args.options.name)}'`,
+        headers: {
+          accept: "application/json;odata.metadata=none"
+        },
+        responseType: 'json'
+      };
+      const response: any = await request.get(requestOptions);
+
+      return response.value && response.value.length === 1 ? response.value[0].id : null;
+    }
+
+    return args.options.id as string;
```

Comment:
If we return this right after the function starts, we can get rid of some indentation for the name.